packages/theme/src/color-ramps/test/index.test.ts

```diff
@@ -15,7 +15,7 @@ const lStops = [ 100, 90, 80, 70, 60, 50, 40, 30, 20, 10 ];
 const sStops = [ 100, 80, 60, 40, 20, 0 ];
 const hstops = [ 0, 60, 120, 180, 240, 300 ];
 
-describe.skip( 'buildRamps', () => {
```

Comment:
Let's see if, by using `colorjs.io`'s internal rounding, we can get the tests to pass in CI and therefore close https://github.com/WordPress/gutenberg/issues/73726